src/Illuminate/Foundation/Providers/ArtisanServiceProvider.php

```diff
@@ -352,6 +354,18 @@ protected function registerDbCommand()
         $this->app->singleton(DbCommand::class);
     }
 
+    /**
+     * Register the command.
+     *
+     * @return void
+     */
+    protected function registerDbPruneCommand()
+    {
+        $this->app->singleton('command.db.prune', function ($app) {
```

Comment:
```suggestion
        $this->app->scoped('command.db.prune', function ($app) {
```
Just some Octane love ❤️

Reply:
There is no problem on having this singleton in Octane context.